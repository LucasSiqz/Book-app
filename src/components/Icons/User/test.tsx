import { render } from 'utils/test-utils'

import User from '.'

describe('<UserIcon />', () => {
  it('should render correctly', () => {
    const { container } = render(<User />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
