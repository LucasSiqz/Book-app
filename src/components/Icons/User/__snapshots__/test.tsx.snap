// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserIcon /> should render correctly 1`] = `
<svg
  fill="none"
  height="19"
  viewBox="0 0 18 19"
  width="18"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M17 18V16C17 13.7909 15.2091 12 13 12H5C2.79086 12 1 13.7909 1 16V18"
    stroke="#BFBEBF"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
  />
  <circle
    cx="9"
    cy="4"
    r="4"
    stroke="#BFBEBF"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
  />
</svg>
`
